.circleci/config.yml

```diff
@@ -1,251 +1,7 @@
-      command: |
-        echo "Comparing with origin/master"
-        git reset --hard origin/master
-        echo "PYVER=${PYVER}"
-        echo 'export PYTHONPATH=$PYTHONPATH:.' >> $BASH_ENV
-        echo 'export BENCHMARK_NPROC=2' >> $BASH_ENV
-        source $BASH_ENV
-        # Max 4 threads to not overtax CI.
-        ${PYTHON} setup.py build_ext --inplace -- -DCOVERAGE=ON -- -j 4
-        git checkout "${CIRCLE_SHA1}" -- benchmarks/
-        BENCHSCR="benchmarks/benchmarker.py"
-        echo "Running benchmark on origin/master"
-        ${PYTHON} "$BENCHSCR" run
-        ${PYTHON} "$BENCHSCR" compare origin/master "${CIRCLE_SHA1}"
-
-  store_benchmarks: &store_benchmarks
-    store_artifacts:
-      path: benchmarks/reports
-      destination: benchmarks-reports
-
-  load_check_style: &load_check_style
-    working_directory: ~/ci/freud
-    steps:
-      - *load_code
-      - *update_submodules
-      - *get_pre_commit_requirements
-      - *pre-test-checks
-
-  build_and_test_windows: &build_and_test_windows
-    executor:
-      name: win/default
-      shell: bash.exe
-    steps:
-      - *load_code
-      - *update_submodules
-      - *get_conda_requirements
-      - *build_windows
-      - *test_cov
-      - *store
-
-  build_and_benchmark: &build_and_benchmark
-    steps:
-      - *load_code
-      - *update_submodules
-      - *get_requirements
-      - *build
-      - *benchmark
-      - *comparison
-      - *store_benchmarks
-
 jobs:
-  check-style:
-    docker:
-      - image: glotzerlab/ci:2020.10-clang11_py39
-    environment:
-      PYVER: "3.9"
-      PYTHON: "python3.9"
-    <<: *load_check_style
-
-  windows-python-310:
-    environment:
-      PYVER: "3.10"
-      <<: *windows_env
-    <<: *build_and_test_windows
-
-  windows-python-39:
-    environment:
-      PYVER: "3.9"
-      <<: *windows_env
-    <<: *build_and_test_windows
-
-  windows-python-38:
-    environment:
-      PYVER: "3.8"
-      <<: *windows_env
-    <<: *build_and_test_windows
-
-  windows-python-37:
-    environment:
-      PYVER: "3.7"
-      DEPENDENCIES: "COMPATIBLE"
-      <<: *windows_env
-    <<: *build_and_test_windows
-
-  benchmarks:
+  build:
     docker:
       - image: glotzerlab/ci:2020.01-py38
-    working_directory: ~/ci/freud
-    environment:
-      PYVER: "3.8"
-      PYTHON: "python3.8"
-    <<: *build_and_benchmark
-
-workflows:
-  version: 2
-  test:
-    jobs:
-      - check-style
-      - windows-python-310
-      - windows-python-39:
-          requires:
-            - check-style
-            - windows-python-310
-      - windows-python-38:
-          requires:
-            - check-style
-            - windows-python-310
-      - windows-python-37:
-          requires:
-            - check-style
-            - windows-python-310
-# Disabled benchmarks on CI - they currently take too much work
-#     - benchmarks:
-#         requires:
-#           - linux-python-38
+    steps:
+      - run: echo "Nothing to do."
```

Comment:
Just delete this file if it doesn't do anything.

Reply:
For some reason, circleci still tries to run even if I remove the config file and it shows up on github as a failed test. The dummy makes sure we can still get the coveted green check mark.

Reply:
Ah. That just takes some fixing on the GitHub / CircleCI website side of things. Go ahead and delete it, and let me know if you want help fixing that.